src/app/login/login.component.ts

```diff
@@ -6,6 +6,7 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./login.component.css']
 })
 export class LoginComponent implements OnInit {
+	login:any = {userName:'' , password:''};
```

Comment:
every time you declare a variable declare it as empty like this
`	login:any = {};